packages/app/src/systems/Asset/services/convert-asset.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  ASSET_ENDPOINTS,
+  DEFAULT_ASSET_ENDPOINT,
+} from '~/systems/Asset/constants';
+
+interface ConvertAssetResponse {
+  amount: `${string}`;
+}
+
+export async function convertAsset(
+  chainId: number | undefined,
+  assetId: string,
+  amount: string
+) {
+  try {
+    const endpoint =
+      (chainId && ASSET_ENDPOINTS[chainId.toString()]) ||
+      DEFAULT_ASSET_ENDPOINT;
+    if (!endpoint) return;
+    const response = await fetch(`${endpoint.url}/convert_rate`, {
+      method: 'POST',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: JSON.stringify({
+        assetId,
+        amount,
+      }),
+    });
+    const jsonResponse = (await response.json()) as ConvertAssetResponse;
+    // Fix a conversion error from the backend
+    if (jsonResponse.amount === '$0') return { amount: '$0.00' };
+    return jsonResponse;
+  } catch (_) {
+    return { amount: '$0.00' };
+  }
+}
```

Comment:
do we need this call if the rate already comes in the asset and we can calculate on front end?

Reply:
We're avoiding making the conversion in the frontend and trying to centralize that in the backend via the endpoint, both here and on explorer. 
cc: @rodrigobranas

Reply:
we should do it in FE instead of doing many requests for a information we already have. just needs to use `bn` correctly